cmd/crowdsec-cli/main.go

```diff
@@ -100,7 +103,7 @@ API interaction:
 	rootCmd.AddCommand(cmdVersion)
 
 	rootCmd.PersistentFlags().StringVarP(&ConfigFilePath, "config", "c", "/etc/crowdsec/config/default.yaml", "path to crowdsec config file")
-	rootCmd.PersistentFlags().StringVar(&OutputFormat, "output", "o", "Output format : human, json, raw.")
+	rootCmd.PersistentFlags().StringVarP(&OutputFormat, "output", "o", "", "Output format : human, json, raw.")
```

Comment:
You can Add default string `human` to avoid this code :
`rootCmd.PersistentFlags().StringVarP(&OutputFormat, "output", "o", "human", "Output format : human, json, raw.")`

```
	if csConfig.Cscli.Output == "" {
		csConfig.Cscli.Output = "human"
	}
```

Reply:
@erenJag no else it's never going to take into account what was in the configuration file

Reply:
oh yes my bad.